public/app/plugins/panel/canvas/editor/element/PlacementEditor.tsx

```diff
@@ -121,7 +121,7 @@ export function PlacementEditor({ item }: Props) {
         <>
           {places.map((p) => {
             const v = placement![p];
-            if (v == null) {
+            if (v == null && p !== 'rotation') {
```

Comment:
maybe a better alternative here is to default rotation to 0 in element options so it exists on every element? 🤔